app/services/jokes.js

```diff
@@ -0,0 +1,23 @@
+const axios = require('axios');
+
+const { externalServiceError } = require('../errors');
+const {
+  common: {
+    geekJokesApi: { baseUrl }
+  }
+} = require('../../config');
+
+exports.getRandomJoke = () =>
+  axios
+    .get(baseUrl)
+    .then(response => {
+      if (response.data) return response.data.joke;
```

Comment:
Whoever calls this function, could get an undefined and must do the check (this could result in multiple lines of code repeated everywhere). Maybe you should check that here and throw an exception, too.